crates/shared/src/order_quoting.rs

```diff
@@ -84,13 +84,9 @@ pub struct Quote {
     /// quotes computed with `SellAmount::BeforeFee` (specifically, it will be
     /// scaled down to account for the computed `fee_amount`).
     pub buy_amount: U256,
-    /// The final minimum subsidized fee amount for any order created for this
-    /// quote. The fee is denoted in the sell token.
+    /// The final fee amount for any order created for this quote. The fee is
```

Comment:
nit: `final` implies that this value gets adjusted multiple times which is no longer the case.
```suggestion
    /// The fee amount for any order created for this quote. The fee is
```